docs/it-manual/sre-playbook/terraform-deploy-system.md

```diff
@@ -1,12 +1,14 @@
 # Terraform deploy system
 
+Terraform - is an infrastructure as code tool that lets you define both cloud and on-prem resources in human-readable configuration files that you can version, reuse, and share. Civiform provides Terraform configuration files that allow to deploy Civiform on Azure and AWS. Knowledge of Terraform is not required to run them but here is the high-level [Terraform overview](https://www.terraform.io/intro) nevertheless. 
+
 ## Setup
 
 ### Outside configuration
-You will need couple of values that are configured outside of Civiform before you start the setup.
-- [ ] Admin auth client_id, client_secret, and discovery_uri. See [setting up Azure AD for an example](#setting-up-azure-ad)
-- [ ] Applicant auth client_id, client_secret, and discovery_uri. See setting up the [Authentication Providers](../../contributor-guide/developer-guide/authentication-providers.md)
-- [ ] Domain name for your deployment
+You will need couple of values that are configured outside of Civiform before you start the setup. Some of the steps are optional meaning that you can bring up staging environment and get app working, but they need to be completed for production setup.
```

Comment:
I'm pretty sure the app will not start if these values are empty. discovery_uri need to be a valid url.

Reply:
Well. We have default dummy values in `civiform_config.example.sh`. You don't have to change them with your own real discovery uri so server can still start. Maybe instead we should have default config with no auth at all (have it disabled). And enabling auth would be a separate step, after you got staging running.